src/main/resources/application.yaml

```diff
@@ -38,6 +38,11 @@ form-flow:
       smarty_auth_id: ${SMARTY_AUTH_ID}
       smarty_auth_token: ${SMARTY_AUTH_TOKEN}
       smarty_url: "https://us-street.api.smartystreets.com/street-address"
+  email-client:
```

Comment:
@vrajmohan - does this configuration only apply to the production app? If so, please disregard this comment: https://github.com/codeforamerica/mdpilot/pull/40/files#r1483722700

Reply:
The configuration applies to all environments but relies on the `MAILGUN_KEY` env var being set. The email will not be sent if `MAILGUN_KEY` is incorrect or not set.